README.md

```diff
@@ -1 +1,163 @@
+
+
+Note that below file paths are based on assumption that all repositories are cloned in same folder.
+
+
+2. Make sure you meet the prerequisites mentioned in both frontend and backend repositories.
+
+3. To run the whole dev setup, you will need to run commands in multiple terminal tabs.
+
+[TAB 1] Insights Proxy TAB -
+Follow steps from [here](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/welcome/quickStart/DOC.md#proxy)
+i.e
+
+```
+cd ../insights-proxy
+npm install
+sudo bash scripts/patch-etc-hosts.sh
+bash scripts/update.sh
+
+```
+
+[TAB 2] Insights Chrome TAB -
+Wait for insights-proxy to finish update script then follow steps from [here](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/welcome/quickStart/DOC.md#chrome)
+i.e
+
+```
+cd ../insights-chrome
+npm install
+npm run build
+Open new tab [TAB 3]: ie. for ros-frontend
+cd ../ros-frontend
+Make sure current directory is `ros-frontend`
+(In case of local ros-backend server) Add route change in the file `ros-frontend/profiles/local-frontend.js` after `routes[`/apps/${APP_ID}`]` i.e
+
+    routes['/api/ros/v0']                = { host: 'http://localhost:8000' };
+    routes['/api/inventory/v1']          = { host: 'http://localhost:8001' };
+
+```
+
+Go to [TAB 1] :
+Make sure current directory is `insights-proxy`
+Run command -
+
+```
+SPANDX_CONFIG=../ros-frontend/profiles/local-frontend.js sh scripts/run.sh
+
+```
+
+[TAB 3] ROS Frontend TAB -
+Wait till the above steps finish their execution.
+
+```
+cd ../ros-frontend
+npm install
+npm run start
+```
+
+4. Follow the steps mentioned under the [ros-backend](https://github.com/RedHatInsights/ros-backend) repository for setting up local backend server.
+
+5. Once both frontend & backend setup is up and running, go to browser and access it using `https://{ci/qa}.foo.redhat.com:1337/insights/ros` link.
+
+
+## Running the Tests
+
+
+Tests can locally be executed with:
+
+```
+npm run test
+```
+
+Few tests are [Snapshot tests](https://jestjs.io/docs/snapshot-testing) used to verify that current test output matches a snapshot taken before. If these changes are legitimate, the snapshots need to be updated with:
+
+```
+npm run test -- -u
+```
+
+
+
+## Running the lint
+
+```
+// to run lint
+npm run lint
+
+// to fix lint errors
+npm run lint:js:fix
+
+```
+
+## Deployment
+
+Please refer this [link](https://clouddot.pages.redhat.com/docs/dev/getting-started/deploying-frontend.html)
+
+
+## Documentation
+
+### Technology
+
+* [React](https://reactjs.org/)
+* [Jest](https://jestjs.io/)
+
+### Major Dependencies used in the Project
+
+- [Insights-Proxy](#insights-proxy)
+- [Spandx](#spandx)
+- [Insights-Chrome](#insights-chrome)
+- [Patternfly](#patternfly)
+- [Frontend-Components](#frontend-components)
+
+### Insights-Proxy
+
+[insights-proxy](https://github.com/RedHatInsights/insights-proxy) - Proxy for the insightsfrontend container
```

Comment:
insightsfrontend, or insights frontend? I'm not sure either way, please double-check.

Reply:
@gnurag updated it to `insights frontend` 👍🏻